shell.h

```diff
@@ -94,18 +94,21 @@ typedef struct cmd_s
  * @key: The key associated with the node.
  * @value: The value associated with the key.
  * @next: Pointer to the next node in the linked list.
+ * @alias: Flag to indicate whether it's an alias (1)
+ * or environment variable (0)
  */
 
 typedef struct s_env
 {
 	char *key;
 	char *value;
-	struct Node *next;
+	struct s_env *next;
+	int alias;
```

Comment:
I think we need to change the name of the struct to `t_pair` and change `alias` to `is_alias` (aliases should have `is_alias=1`).

Reply:
okay

Reply:
check now, commits [fc73322] and [d3f7929]